src/Avatar/Avatar.js

```diff
@@ -22,6 +22,7 @@ function getStyles(props, context) {
       borderRadius: '50%',
       height: size,
       width: size,
+      boxShadow: src ? `inset 0px 0px 0px 1px ${avatar.borderColor}` : 'none',
```

Comment:
Are you sure about the visual impact of this property? It seems to me that it has no impact and that `backgroundColor` seems to be relevant here (I have no idea why).